README.md

```diff
@@ -70,7 +70,7 @@ every :sunday, at: '12pm' do # Use any day of the week or :weekend, :weekday
   runner "Task.do_something_great"
 end
 
-every '0 0 27-31 * *' do
+every '0 0 27-31 * *' do # Note the absence of trailing space
```

Comment:
I don’t think pointing out this edge case in the happy path of documentation is necessary.